### DISCUSSION TITLE
在压测过程中遇到jemalloc 内存利用率过低问题

### DISCUSSION THREAD

**Discussion by jiangshuai377** [cloudwu/skynet#1868] [General]
jmem 
stats.active     1597714432   1523.70 Mb
stats.allocated   320567696    305.72 Mb
stats.mapped     1731465216   1651.25 Mb
stats.resident   1669242880   1591.91 Mb
stats.retained   2662592512   2539.25 Mb

stats.active (1597444096 bytes, 约 1523.44 Mb):
这是当前被应用程序分配并且正在使用的内存量。这些内存已经从操作系统请求并分配给了应用程序，但并不意味着它们都在物理内存中；一部分可能因为最近没有被访问而被换出到交换空间（swap）
stats.allocated (320090856 bytes, 约 305.26 Mb):
表示应用程序通过 jemalloc 分配器请求并且当前正在使用的内存量。这是实际上由应用程序逻辑直接使用的内存大小，它是 stats.active 的一个子集。

压测场景是账号登录、
登录时我们会为每个账号创建一个service用来处理登录 、
登录成功后销毁登录的service 创建一个service来处理这个玩家的业务、
下线时延迟销毁 

云大 有什么排查思路吗 会不会时因为lua虚拟机频繁创建销毁导致的

**Comment by cloudwu** [comment]
非 bug 不要提 issue 。

**Comment by cloudwu** [comment]
我不认为 1.5G 内存对一个服务器来说过多。可能你的浏览器都比它用的多。

你应该改进你的压测程序更接近真实情况，比如 64G / 128G 内存时的状态。

**Reply by jiangshuai377** [reply]
我丰富测试用例 看一下后续
